src/components/picker/__tests__/index.spec.tsx

```diff
@@ -100,122 +96,191 @@ describe('Picker', () => {
       });
     });
 
-    it('Test onDismiss', () => {
-      const driver = getDriver({
-        pickerModalProps: {
-          onDismiss
-        }
-      });
-
-      expect(driver.isOpen()).toBeFalsy();
+    it('Test onPress', () => {
+      const driver = getDriver({onPress});
       driver.open();
       expect(driver.isOpen()).toBeTruthy();
       driver.cancel();
       expect(driver.isOpen()).toBeFalsy();
-      expect(onDismiss).toHaveBeenCalledTimes(2); // TODO: this should be 1
+      expect(onPress).toHaveBeenCalled();
+    });
```

Comment:
Shouldn't this be only this:
```
const driver = getDriver({onPress});
driver.open();
expect(onPress).toHaveBeenCalled();
```

Reply:
you are right

Reply:
I don't see a fix for this; IMO it should be:
```
it('Test onPress', () => {
  const driver = getDriver({onPress});
  driver.open();
  expect(onPress).toHaveBeenCalled();
});

```